docs/extensions.md

```diff
@@ -148,6 +148,40 @@ To get real-time data for providers who aren't using Open Data, follow the instr
 
 ---
 
+## Unlocking geocoding providers
+
+To get results from geocoding data for providers who aren't using Open Data when using `geocoding.json` endpoint, follow the instructions for the provider below. If you want to use a provider that hasn't yet been connected to our system, please get in touch with our team [by mail](mailto:api@skedgo.com) or on [Slack](http://slack.tripgo.com/) (by [self-invite](http://slackin.tripgo.com/)).
+
+### Google Places 🌎
+
+1. Read the [Places API Policies](https://developers.google.com/places/web-service/policies) and make sure you comply with them in your app - in particular, that you are using a Google Map.
+2. Sign up to the [Google Places API](https://developers.google.com/places/web-service/get-api-key)
+3. Enter your `Google Places API key` in your [application credentials](https://tripgo.3scale.net/admin/applications). 
+4. Geocoding results will then come augmented with Google Places's API through the TripGo API for your API key (after at most 5 minutes).
```

Comment:
Should add a "Add `allowGoogle=true` to your geocoding requests, and then geocoding results will..."

(And same for the others)